config/schedule.rb

```diff
@@ -16,8 +16,10 @@
   runner 'WorkerHealthChecker.enqueue_dummy_jobs'
 end
 
-mail_batch = Whenever.seconds(Figaro.env.usps_mail_batch_hours.to_i, :hours)
+if Figaro.env.enable_usps_verification == 'true'
```

Comment:
Good point, I guess if `environment.rb` is loaded we can just use that. Will fix!